misc/lint/copyright.awk

```diff
@@ -9,13 +9,18 @@
 #
 # copyright.awk — checks file for missing copyright header.
 
+function err(s)
+{
+    print "lint: \033[31merror:\033[0m copyright: " s > "/dev/stderr"
+}
+
 function done()
 {
     if (!copyright) {
-        print "lint: copyright: " FILENAME " is missing copyright header" > "/dev/stderr"
+        err(FILENAME " is missing copyright header")
         exit 1
     } else if (copyright !~ /Copyright Materialize, Inc./) {
-        print "lint: copyright: " FILENAME " has malformatted copyright header" > "/dev/stderr"
+        err(FILENAME " has malformatted copyright header")
```

Comment:
If this includes the `copyright` variable it will make it more obvious what's wrong, especially if the string `Copyright` appears in the first comment multiple times:

```suggestion
        err(FILENAME " has malformatted copyright header: " copyright)
```

Reply:
I tried that, but it results in really long error messages like "lint: error: copyright: foo/bar/baz has malformatted copyright header: # Copyright SOME OTHER CO, Inc. All rights reserved" which I found hard to parse. Adjusted to print a hint mentioning the line number and the exact requirement.